### PR TITLE
fix: removed exponent e from input fields

### PR DESCRIPTION
**Pull request by zarej**

### CODE REVIEW

File: src/app/util/helper.ts [hunk 1]
```diff
@@ -27,24 +27,24 @@ export const urlTo = (path: string, params?: UrlParamType) => {
...
     .floor()
-    .toFixed()
-    .toString();
+    .toFixed();
 };
 
-export const formatDecimalsFromToken = (base: number, decimals: string) => {
-  return toFixedNumber(new Decimal(base).dividedBy(Math.pow(10, parseFloat(decimals))).toNumber());
+export const formatDecimalsFromToken = (base: Decimal.Value, decimals: string) => {
+  // console.log("formatDecimalsFromToken", base, decimals)
```
Comment (ngrujicic) [inline src/app/util/helper.ts]: remove console.log

**Comment by ngrujicic** [comment]
projekat se ne bilduje, imas greske ovde:
Screenshot 2023-12-04 at 13 19 24

File: src/app/util/helper.ts [hunk 2]
```diff
@@ -93,3 +93,35 @@ export const toFixedNumber = (number: number) => {
...
+ * @param input format like "110.4089 µKSM", "1.9200 mWND" or "0.001919 WND"
+ * @returns converted value in the base unit (e.g. 0.0000001104089, 0.00192)
+ */
+export const convertToBaseUnit = (input: string): Decimal => {
+  // Regular expression to extract the number, optionally the unit (m or µ), followed by the currency type
+  const regex = /(\d+\.?\d*)\s*([mµ])?\w+/;
+  const match = input.match(regex);
+
+  if (!match) {
+    console.log("Invalid input format", input);
```
Comment (ngrujicic) [inline src/app/util/helper.ts]: remove console.log

File: src/app/util/helper.ts [hunk 3]
```diff
@@ -93,3 +93,35 @@ export const toFixedNumber = (number: number) => {
...
+    return new Decimal(0);
+  }
+
+  const [, rawValue, unit] = match;
```
Comment (ngrujicic) [inline src/app/util/helper.ts:110]: what is this [, ?
i think this should be removed

File: src/components/organism/AddPoolLiquidity/index.tsx [hunk 4]
```diff
@@ -281,6 +277,7 @@ const AddPoolLiquidity = ({ tokenBId }: AddPoolLiquidityProps) => {
   const setSelectedTokenBValue = (value: string) => {
     setInputEdited({ inputType: InputEditedType.exactOut });
     if (slippageValue && value !== "") {
+      // value = new Decimal(value).toFixed();
```
Comment (ngrujicic) [inline src/components/organism/AddPoolLiquidity/index.tsx]: remove this comment

Comment (zarej) [reply src/components/organism/AddPoolLiquidity/index.tsx]: uncommented, it was commented because of testing

**Review by ngrujicic — APPROVED** [review]

**Pull request merged**